src/main/java/com/github/monster860/fastdmm/objtree/ObjectTreeParser.java

```diff
@@ -122,9 +122,9 @@ public void doParse(BufferedReader br, Path currentFile, boolean isMainFile) thr
                 line = line.trim();
                 if (line.startsWith("#include")) {
                     String path = "";
-                    String includeData = line.split(" ");
+                    String includeData = line.split(" ")[1];
```

Comment:
This would throw an exception if the array didn't have a record at [1].

Reply:
I believe that it cant occur with that because if you see at line 121 it has to start with # and any whitespace then gets trimmed at line 122 before it come to this point.